nmdc_runtime/api/endpoints/find.py

```diff
@@ -176,44 +176,46 @@ def find_data_object_by_id(
 
 
 @router.get(
-    "/activities",
+    "/workflow_executions",
     response_model=FindResponse,
     response_model_exclude_unset=True,
 )
-def find_activities(
+def find_workflow_executions(
     req: FindRequest = Depends(),
     mdb: MongoDatabase = Depends(get_mongo_db),
 ):
+    # TODO: Update description to be accurate wrt Berkeley schema
     """
-    The GET /activities endpoint is a general way to fetch metadata about various activities (e.g. metagenome assembly,
+    The GET /workflow_executions endpoint is a general way to fetch metadata about various workflow_executions (e.g. metagenome assembly,
     natural organic matter analysis, library preparation, etc.). Any "slot" (a.k.a. attribute) for
-    [WorkflowExecutionActivity](https://microbiomedata.github.io/nmdc-schema/WorkflowExecutionActivity/)
+    [WorkflowExecution](https://microbiomedata.github.io/nmdc-schema/WorkflowExecution/)
     or [PlannedProcess](https://microbiomedata.github.io/nmdc-schema/PlannedProcess/) classes may be used in the filter
-    and sort parameters, including attributes of subclasses of *WorkflowExecutionActivity* and *PlannedProcess*.
+    and sort parameters, including attributes of subclasses of *WorkflowExecution* and *PlannedProcess*.
 
-    For example, attributes used in subclasses such as MetabolomicsAnalysisActivity (subclass of *WorkflowExecutionActivity*)
+    For example, attributes used in subclasses such as MetabolomicsAnalysis (subclass of *WorkflowExecution*)
     or [Extraction](https://microbiomedata.github.io/nmdc-schema/Extraction/) (subclass of *PlannedProcess*),
     can be used as input criteria for the filter and sort parameters of this endpoint.
     """
     return find_resources_spanning(req, mdb, activity_collection_names(mdb))
```

Comment:
Want to refactor name of `activity_collection_names` function as well, yes? `workflow_execution_collection_names`? Or I forget if there's only one collection in the new schema. Even if no change now, please leave comment about this concern here in the code.

Reply:
There is one collection for all WorkflowExecution documents in berkeley schema, [workflow_execution_set](https://microbiomedata.github.io/berkeley-schema-fy24/workflow_execution_set/)